src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -270,17 +350,26 @@ def _fetch_assemblies_to_upload(
         ]
 
         if len(assembly_file_list) != 1:
-            raise ValueError(f"One and only one assembly file that ends with {upload_file_ext} "
-                    f"must be present in {assembly_dir} directory")
+            raise ValueError(
+                f"One and only one assembly file that ends with {upload_file_ext} "
+                f"must be present in {assembly_dir} directory"
+            )
 
         count += 1
         assembly_name = assembly_file_list[0]
```

Comment:
Assembly: GCF_000968435.1_ASM96843v1_genomic.fna.gz
Genome: GCF_000968435.1_ASM96843v1_genomic.gbff.gz

I prefer to keep that extension as it helps to distinguish what we are uploading and not overwriting stuff if they (both assembly and genome) are in the same workspace.

Reply:
👍 yea. not a big deal. But Genome and Assembly objs can have the same name since their type is different.

Reply:
Not sure I understand what you're saying, but if you have Assembly `xyz` in a workspace and then save Genome `xyz` it'll save over the assembly and make a new version

Reply:
oh really. I thought objs can have the same name if they are different type. Maybe I am wrong.

Reply:
Nope, otherwise you'd have to specify the type as well as the ws id/name and object name when fetching an object since just the ws and the obj name wouldn't necessarily uniquely ID an object